CRM/Volunteer/BAO/Project.php

```diff
@@ -231,11 +239,14 @@ static function retrieveByID($id) {
    * @access public
    */
   public static function dataExists($params) {
```

Comment:
Fair enough. That method has been around for a while, though -- pretty sure it was borrowed from some core class -- and it was outside the scope of what I was doing to rename it. Will keep in mind as other refactoring happens.